### DISCUSSION TITLE
[REQUEST]: Save and Log safety score data

### DISCUSSION THREAD

**Discussion by matze19999** [teslamate-org/teslamate#2003] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

Safety score data is available over the tesla api.
Any chance to save and log this data and watch in grafana?

### Relevant log output

```shell
no output
```

### Type of installation

Docker

### Version

1.24.1

**Comment by nicholashead** [comment]
This appears to be a dupe of https://github.com/adriankumpf/teslamate/discussions/1997
